Tests/PubNubTests/Networking/Routers/ChannelGroupEndpointTests.swift

```diff
@@ -284,36 +282,36 @@ final class ChannelGroupsEndpointTests: XCTestCase {
         case .success:
           XCTFail("Add Channel request should fail")
         case let .failure(error):
-          XCTAssertEqual(error.pubNubError, PubNubError(reason: .invalidCharacter))
+          XCTAssertEqual(error.pubNubError, PubNubError(.invalidCharacter))
         }
         expectation.fulfill()
       }
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
 
-  // MARK: - Group Channel Remove
+// MARK: - Remove Channel from Group
 
-  func testGroupChannelsRemove_Endpoint() {
-    let endpoint = Endpoint.removeChannelsForGroup(group: testGroupName, channels: testChannels)
+extension ChannelGroupsRouterTests {
+  func test_RemoveChannelsForGroup_Router() {
+    let router = ChannelGroupsRouter(.removeChannelsForGroup(group: testGroupName, channels: testChannels), configuration: config)
 
-    XCTAssertEqual(endpoint.description, "Group Channels Remove")
-    XCTAssertEqual(endpoint.category, .removeChannelsForGroup)
-    XCTAssertEqual(endpoint.operationCategory, .channelGroup)
-    XCTAssertNil(endpoint.validationError)
+    XCTAssertEqual(router.endpoint.description, "Group Channels Remove")
+    XCTAssertEqual(router.category, "Group Channels Remove")
+    XCTAssertEqual(router.service, .channelGroup)
   }
 
-  func testGroupChannelRemove_Endpoint_ValidationError() {
-    let endpoint = Endpoint.removeChannelsForGroup(group: "", channels: [])
+  func test_RemoveChannelsForGroup_Router_ValidationError() {
+    let router = ChannelGroupsRouter(.removeChannelsForGroup(group: "", channels: testChannels), configuration: config)
 
-    XCTAssertNotEqual(endpoint.validationError?.pubNubError, PubNubError(.invalidEndpointType, endpoint: endpoint))
-  }
+    XCTAssertEqual(router.validationError?.pubNubError?.details.first,
+                   ErrorDescription.emptyGroupString)
 
-  func testGroupChannelRemove_Endpoint_AssociatedValues() {
-    let endpoint = Endpoint.removeChannelsForGroup(group: "GroupName", channels: ["ChannelName"])
+    let emptyChannels = ChannelGroupsRouter(.removeChannelsForGroup(group: testGroupName, channels: []), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 127 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)